packages/next/src/server/web/spec-extension/image-response.ts

```diff
@@ -1,52 +1,5 @@
-export class ImageResponse extends Response {
-  public static displayName = 'NextImageResponse'
-  constructor(
-    ...args: ConstructorParameters<
-      typeof import('next/dist/compiled/@vercel/og').ImageResponse
-    >
-  ) {
-    const readable = new ReadableStream({
-      async start(controller) {
-        const OGImageResponse: typeof import('next/dist/compiled/@vercel/og').ImageResponse =
-          // So far we have to manually determine which build to use,
-          // as the auto resolving is not working
-          (
-            await import(
-              process.env.NEXT_RUNTIME === 'edge'
-                ? 'next/dist/compiled/@vercel/og/index.edge.js'
-                : 'next/dist/compiled/@vercel/og/index.node.js'
-            )
-          ).ImageResponse
-        const imageResponse = new OGImageResponse(...args) as Response
-
-        if (!imageResponse.body) {
-          return controller.close()
-        }
-
-        const reader = imageResponse.body!.getReader()
-        while (true) {
-          const { done, value } = await reader.read()
-          if (done) {
-            return controller.close()
-          }
-          controller.enqueue(value)
-        }
-      },
-    })
-
-    const options = args[1] || {}
-
-    super(readable, {
-      headers: {
-        'content-type': 'image/png',
-        'cache-control':
-          process.env.NODE_ENV === 'development'
-            ? 'no-cache, no-store'
-            : 'public, immutable, no-transform, max-age=31536000',
-        ...options.headers,
-      },
-      status: options.status,
-      statusText: options.statusText,
-    })
-  }
+export function ImageResponse() {
+  throw new Error(
+    'ImageResponse export is moved to next/og since Next.js 14, please import from "next/og" instead'
```

Comment:
```suggestion
    'ImageResponse has moved from "next/server" to "next/og" since Next.js 14, please import from "next/og" instead'
```

This will help the user grep "next/server" usage